kqueen/engines/openstack_kubespray.py

```diff
@@ -700,7 +724,14 @@ def _get_userdata(self):
             "package_update": True,
             "packages": ["python"],
             "ssh_authorized_keys": [self.extra_ssh_key],
+            "write_files": [
+                {
+                    "content": '{"bip": "10.13.0.1/16"}',
```

Comment:
maybe define it as kwarg and get it from create-cluster page ? imo it will be much better for future deployments

Reply:
and also need to specify this feature in docs.rst

Reply:
See line 744. We also need to do the same at least with `kube_service_addresses` and `kube_pods_subnet`

Reply:
yep, i know, but imo it should be different changes, we may define docker networks in this PR, coz it directly depends on main PR idea (bugfix) and provide separate PR with kube-networks. 2nd case - is docker networks defining and kube-networks defining  doesnot depends on each other and should be tested separately